cmd/libsnap-confine-private/apparmor-support.c

```diff
@@ -65,9 +69,13 @@ void sc_init_apparmor_support(struct sc_apparmor *apparmor)
 			// never happen so likely someone is trying to
```

Comment:
this comment now doesn't appear to match the behaviour - previously we would fall-through to the `die()` in the `default` case, but with the proposed change the `break` means we exit the `switch` - I feel like the previous behaviour should be retained where we `die()` on both `EPERM` and `EACCESS` - can you please add an explicit `die()` above this `break;` within the `EACCESS` case?

Reply:
This is now done. I've kept EPERM == EACCESS but made everything else an explicit `die`.